src/main/java/frc/robot/commands/DropPlacer.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.commands;
+
+import com.spikes2212.command.genericsubsystem.commands.MoveGenericSubsystem;
+import com.spikes2212.dashboard.RootNamespace;
+import edu.wpi.first.wpilibj.Timer;
+import frc.robot.subsystems.ClimberPlacer;
+
+import java.util.function.Supplier;
+
+public class DropPlacer extends MoveGenericSubsystem {
+
+    private final ClimberPlacer placer;
+
+    private static final RootNamespace rootNamespace = new RootNamespace("drop placers");
+    public static final Supplier<Double> initialWaitTime = rootNamespace.addConstantDouble("initial wait time", 0.25);
```

Comment:
convention A-I\a2

Reply:
Convention Yotam-a12345678: write the convention in the comment so I don't need to look it up in the file.